examples/aligning_aia_with_eis_maps.py

```diff
@@ -0,0 +1,68 @@
+"""
+=====================
+Coaligning EIS to AIA
+=====================
+
+This example shows how to EISA data to AIA using cross-correlation which is implemented as the "match_template" method.
+"""
+
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+from sunpy.net import Fido ,attrs as a
+import sunpy.map
+from sunkit_image.coalignment import coalignment
+from sunkit_image.data.test import get_test_filepath
+
+###################################################################################
+# Firstly, us acquire the EIS and AIA data we need for this example.
+
+eis_map = sunpy.map.Map(get_test_filepath("eis_20140108_095727.fe_12_195_119.2c-0.int.fits"))
+eis_map.plot()
```

Comment:
Use these settings when plotting the EIS images:

https://github.com/USNavalResearchLaboratory/eispac/blob/58782ad348c2c644973cbbb4c575925484fafcc0/eispac/core/eismap.py#L61-L66